### ISSUE TITLE
Elevated urine aconitic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6032]
**Preferred term label:** Elevated urine aconitic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of aconitic acid in the urine is above the normal range., PMID: 30830355

**Parent term (use hpo.jax.org/app)** Elevated urinary carboxylic acid HP:0040156

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033101
